src/BlocklyInterface/BlocklyEditor.tsx

```diff
@@ -138,6 +162,10 @@ export const BlocklyEditor: FunctionComponent = () => {
         executing ? "Your program cannot be changed until you stop it" : ""
       }
     >
+      <img
```

Comment:
Fluent UI has button components that display icons. Could you check https://developer.microsoft.com/en-us/fluentui#/controls/web/button and https://developer.microsoft.com/en-us/fluentui#/controls/web/icon for an appropriate one?

Reply:
Nothing nice found which matches in a way that it's intuitive.
I think we can delay beautifying and choosing correct icons until we have an idea about the UI should look like overall.

Reply:
> Nothing nice found which matches in a way that it's intuitive.

A pretty common pattern is to use a Chevron left / right to collapse to the left, expand to the right.

Reply:
> I think we can delay beautifying and choosing correct icons until we have an idea about the UI should look like overall.

Maybe we can set up a meeting with who ever is interested to make some mock ups for how it should look going forward so we don't have a lot of refactoring later on and have an idea of what we're working on.

Reply:
Very good idea. We need a concept of how the UI should look like overall at the end.
If the icon is just the last thing which is open from this PR then this shouldn't be a blocker to go forward with this PR, is it?

Reply:
> If the icon is just the last thing which is open from this PR then this shouldn't be a blocker to go forward with this PR, is it?

Can you make a follow up issue for it?

Reply:
I created an issue for an overall UI design